be/src/vec/exec/scan/vfile_scanner.cpp

```diff
@@ -785,9 +785,17 @@ Status VFileScanner::_get_next_reader() {
             break;
         }
         case TFileFormatType::FORMAT_PARQUET: {
+            static const cctz::time_zone utc0 = cctz::utc_time_zone();
+            cctz::time_zone* tz;
+            if (range.__isset.table_format_params &&
+                range.table_format_params.table_format_type == "paimon") {
+                tz = const_cast<cctz::time_zone*>(&utc0);
```

Comment:
Why paimon use utc?
add comment in code

Reply:
done